packages/engine/src/api-handler.ts

```diff
@@ -11,8 +11,16 @@ import { StatusCodes } from 'http-status-codes';
 import { executeEngine } from './engine-executor';
 import { EngineRequest } from './main';
 
+const isLambdaEnvironment = (): boolean => {
+  return process.env.AWS_LAMBDA_FUNCTION_NAME !== undefined;
+};
+
+const LAMBDA_FASTIFY_BODY_LIMIT = 6 * 1024 * 1024; // 6MB
+const DEFAULT_FASTIFY_BODY_LIMIT = 10 * 1024 * 1024; // 10MB
 const app = fastify({
-  bodyLimit: 6 * 1024 * 1024, // 6MB same as engine api-handler & nginx.conf
+  bodyLimit: isLambdaEnvironment()
```

Comment:
use an env var

Reply:
so if it is an existing installation, and we add an env var, will it be active in that installation? Asking because I think the template .env is only copied when you first install.